study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -58,6 +58,7 @@ private void run(String[] args) throws Exception {
         options.addOption(null, "dry-run", false, "run study setup or custom task without saving");
         options.addOption(null, "only-activity", true, "only run activity setup for given activity code");
         options.addOption(null, "only-workflow", false, "only run workflow setup");
+        options.addOption(null, "only-update-workflow", false, "recreate workflow from configuration file");
```

Comment:
Maybe call the new flag `--recreate-workflow`? "update" makes me think that it'll selectively add new workflows or update current ones.

Reply:
Sure. 0a62f2fa